contracts/epochs/FlowEpoch.cdc

```diff
@@ -57,9 +57,36 @@ pub contract FlowEpoch {
         pub case Access
     }
 
+    /// The Epoch Start service event is emitted when the contract transitions
+    /// to a new epoch in the staking auction phase.
+    pub event EpochStart (
```

Comment:
Will this be emitted along with NewEpoch from FlowIDTableStaking?

Reply:
yes, it will be emitted in the same transaction